pkgs/development/libraries/bobcat/default.nix

```diff
@@ -26,7 +26,10 @@ stdenv.mkDerivation rec {
     patchShebangs .
   '';
 
-  buildPhase = ''
+  buildPhase = lib.optionalString (with stdenv.targetPlatform; isAarch64 && isLinux) ''
+    # have to link to static gcc lib on aarch64-linux explicitly
+    NIX_LDFLAGS="$NIX_LDFLAGS -lgcc"
+  '' + ''
```

Comment:
Assuming it works it's much simpler to just set `NIX_LDFLAGS` as a derivation attribute - the derivation will splice it onto any automatically-set `NIX_LDFLAGS` values. grep nixpkgs for how it's used 90% of the time.

Reply:
TIL. Thanks!